RiotSharp/Properties/AssemblyInfo.cs

```diff
@@ -4,14 +4,14 @@
 // General Information about an assembly is controlled through the following
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("RiotSharp")]
-[assembly: AssemblyDescription("C# Wrapper for the Riot Games API")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("BenFradet")]
-[assembly: AssemblyProduct("RiotSharp")]
-[assembly: AssemblyCopyright("Copyright (c) Benjamin Fradet 2013-2017")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
+//[assembly: AssemblyTitle("RiotSharp")]
+//[assembly: AssemblyDescription("C# Wrapper for the Riot Games API")]
+//[assembly: AssemblyConfiguration("")]
+//[assembly: AssemblyCompany("BenFradet")]
+//[assembly: AssemblyProduct("RiotSharp")]
+//[assembly: AssemblyCopyright("Copyright (c) Benjamin Fradet 2013-2017")]
+//[assembly: AssemblyTrademark("")]
+//[assembly: AssemblyCulture("")]
```

Comment:
can you remove the commented lines so we can have a clearer a view of what's been removed?